### ISSUE TITLE
Installation issue - Win64

### ISSUE THREAD

**Issue by yazoo321** [sha3sha3/UE-EasyKafka#2]
Hi,

Is it possible to add a bit more to installation instructions?

The plugin was not by default compatible with my engine (5.1) , so I compiled it using these steps: https://www.youtube.com/watch?v=sC0gnfYzFzU

If I add the suggested lines to the `Build.cs` the build fails without any useful info.
![image](https://user-images.githubusercontent.com/19325521/230043927-b510a973-9f21-4061-b7ae-ce36c7348596.png)

After compiling and adding the plugin into the engine, it's visible from Plugins menu, if I enable it however, I get the error.
![image](https://user-images.githubusercontent.com/19325521/230044147-1a0731dc-266d-4d62-9342-05f5185d113b.png)
![image](https://user-images.githubusercontent.com/19325521/230044228-606f3697-0deb-407d-94ad-cba379b7cbb7.png)

I expected it was failing to load a dll and that seems correct, from logs I find:
```
[2023.04.05-09.42.48:186][  0]LogWindows: Failed to preload 'G:/Unreal/UE_5.1/Engine/Plugins/EasyKafka/Binaries/Win64/UnrealEditor-KafkaConsumer.dll' (GetLastError=126)
[2023.04.05-09.42.48:186][  0]LogWindows:   Missing import: rdkafka.dll
[2023.04.05-09.42.48:186][  0]LogWindows:   Looked in: ../../../Engine/Binaries/Win64
[2023.04.05-09.42.48:186][  0]LogWindows:   Looked in: ../../../../../MyMmoRpg/Binaries/Win64
[2023.04.05-09.42.48:186][  0]LogWindows:   Looked in: G:\MyMmoRpg\Binaries\Win64
......
```

searching the plugin, I can see it's present in: `G:\Unreal\UE_5.1\Engine\Plugins\EasyKafka\Source\ThirdParty\KafkaLib\bin\Win64`

This makes me think there's some steps missing from installation - could you advise ?

As a temporary fix, I am able to copy those dlls over to: `G:\MyMmoRpg\Binaries\Win64` but this is more of a hack.
This allows me to enable the plugin and have the project load now (I will shortly test if it indeed actually works though)

however even with that, I am still not able to add:
```
        PrivateDependencyModuleNames.AddRange(new string[] 
        {
            "EasyKafka",
            "KafkaLib",
            "KafkaConsumer",
            "KafkaProducer"
        });
```
the above still does not compile

[update] even with the 'hack' above, following the suggested BP to publish kafka message leads to a crash:
![image](https://user-images.githubusercontent.com/19325521/230058403-58260529-0d99-4be1-abb3-b2df2edd13eb.png)
![image](https://user-images.githubusercontent.com/19325521/230059296-2ffa43f2-e56b-4cdb-957f-8297679798d8.png)

(not clear if its configuration or misuse issue)

**Comment by sha3sha3** [comment]
Hey @yazoo321 
On paper UBT should copy those DLLs to the Win64 folder, your hack fixed that!
I am working on an update to statically link everything & with UE third party libs, you can test with the Experimental Branch if you are on Windows.
Regarding the crash, if you can attach some logs that migh help us to figure out what is wrong!

**Comment by yazoo321** [comment]
The crash was caused by leaving the **username** and **password** blank, from the screenshot above (part of create producer function). 
my initial test did not have secured kafka, so I am changing that now but its taking a little longer than anticipated
I will update once tests are complete, thanks

**Comment by yazoo321** [comment]
I managed to get it working, there were a couple of issues with silent failures but I found some working configuration
I put together a lot of the steps I needed to do in order to get it working here:
https://unreal-mmo-dev.com/2023/04/18/38-how-to-easily-compile-your-plugin-with-ue-easykafka-example/

feel free to include it in the readme if you like.
thanks

**Comment by sha3sha3** [comment]
Hi

@yazoo321 i will close this issue.
Fixed in commmit c4ae5d36d18cda80c436acef50fbbbb0c8ac91c5.
